css/properties/aspect-ratio.json

```
@@ -0,0 +1,137 @@
+{
+  "css": {
+    "properties": {
+      "aspect-ratio": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/aspect-ratio",
+          "support": {
+            "chrome": {
+              "version_added": "79",
+              "partial_implementation": true,
+              "notes": "Chrome 79 adds internal support only for mapped values"
+            },
+            "chrome_android": {
+              "version_added": "79",
+              "partial_implementation": true,
+              "notes": "Chrome 79 adds internal support only for mapped values"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": [
+              {
+                "version_added": "71",
+                "partial_implementation": true,
+                "notes": "Firefox 71 adds internal support only for mapped values"
+              },
+              {
+                "version_added": "69",
+                "version_removed": "71",
+                "partial_implementation": true,
+                "notes": "Firefox 69 adds internal support only for mapped values",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "layout.css.width-and-height-map-to-aspect-ratio.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              }
+            ],
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "79",
```

Looks like one was missed, but otherwise this looks good to me!

```suggestion
              "version_added": "79",
              "partial_implementation": true,
```